change/@ni-nimble-components-6eb6eef1-eec2-4f6d-a98f-f7182ecdad50.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why do we need a change file for `nimble-components`? I don't see any diffs in its project folder. Or is this related to the `package.json` changes?

Reply:
Originally this was a stale change file (resulting in me starting from my previous branch instead of main) which could be deleted.

Now though, there's a nimble-components change file (type=none) due to the Storybook/documentation update.